src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -635,16 +635,56 @@ class TablePumpDescription:
         )
     )
 
+    # H [m] = P [Pa] * Q [m^3/s] / rho_ref [kg/m^3] / g [m/s^2]
```

Comment:
Perhaps move this do the class docstring, since this will avoid SonarCoud's code smell check ;)

https://sonarcloud.io/project/issues?pullRequest=273&issues=AYQ_HGssxiQdHZqfkiyy&open=AYQ_HGssxiQdHZqfkiyy&id=ESSS_alfasim-sdk

Reply:
As suggest, moved the commented code to the class's docstring.